### PR TITLE
DDBTEAM-442: Small debug fixes to upload queue processor

### PR DESCRIPTION
**Pull request by cableman**
https://jira.itkdev.dk/browse/DDBTEAM-442

### CODE REVIEW

File: src/Command/QueueInsertCommand.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $topic = $input->getOption('topic');
+        $message = $input->getOption('message');
+        $withTestMessage = $input->getOption('with-test-message');
+
+        if ($withTestMessage) {
+            // Fallback message for easy testing.
```
Comment (tuj) [inline src/Command/QueueInsertCommand.php]: Fallback message => Test messages

File: src/Command/QueueInsertCommand.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                    $processMessage->setIdentifier('9788799239535')
+                        ->setOperation(VendorState::UPDATE)
+                        ->setIdentifierType(IdentifierType::ISBN)
+                        ->setVendorId(1)
+                        ->setImageId(1);
+                    $message = JSON::encode($processMessage);
+                    break;
+
+                default:
+                    throw new \RuntimeException('No default message exists to the topic');
```
Comment (tuj) [inline src/Command/QueueInsertCommand.php]: No test message exists for the given topc

File: src/Command/QueueInsertCommand.php (hunk 1 above)
Comment (cableman) [reply src/Command/QueueInsertCommand.php]: fixed

File: src/Command/QueueInsertCommand.php (hunk 2 above)
Comment (cableman) [reply src/Command/QueueInsertCommand.php]: Fixed

**Pull request merged**